lib/emotions.rb

```diff
@@ -8,35 +8,35 @@
 require 'emotions/emotional'
 
 module Emotions
-  def self.configure
+  def configure
```

Comment:
Indeed, too much `self.` to remove :) It's fixed, Travis should go :green_heart:.